### ISSUE TITLE
Flow types used in explicit type arguments in function calls are reported as not being used

### ISSUE THREAD

**Issue by rubennorte** [babel/babel-eslint#644]
Given the following code:

```
type Type = string;

function identity<T>(val: T): T {
  return val;
}

const foo = identity<Type>('foo');
```

`Type` is being reported as not being used by ESLint. Identifiers used inside explicit type arguments in function calls should be visited and not reported as unused.

**Comment by rubennorte** [comment]
This is not fixed yet. We still have to wait for the Babel release and merge #645.

**Comment by TrySound** [comment]
Sorry, it's because one of my transitive dependency wasn't updated. Reinstall helped.
